core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.ts

```diff
@@ -215,4 +219,20 @@ export class DotEditLayoutComponent implements OnInit, OnDestroy {
             {}
         );
     }
+
+    /**
+     * Handle save changes before leave
+     *
+     * @private
+     * @memberof DotEditLayoutComponent
+     */
+    private saveChangesBeforeLeave(): void {
```

Comment:
I think the name is not properly, since it is subscribing a handle rather than doing the actual save, it should be something such as

subscribeOnChangeBeforeLeaveHandler

Reply:
Nice catch!